zerver/migrations/0521_stream_stream_topic_access_group.py

```diff
@@ -0,0 +1,21 @@
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        (
+            "zerver",
+            "0520_attachment_zerver_attachment_realm_create_time",
+        ),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="stream",
+            name="stream_topic_access_group",
+            field=models.ForeignKey(
+                null=True, on_delete=django.db.models.deletion.RESTRICT, to="zerver.namedusergroup"
```

Comment:
The only reason I am writing `NamedUserGroup` as foreign key instead of `UserGroup` is that I want to use to `is_support_stream` function defined [here](c7a8b3c045136acf0dc74b97979f5769632e2351).

With `UserGroup` there will be no `name` property on `stream_topic_access_group` and I will have to do something like this which I think will be a heavier database operation:

```
NamedUserGroup.objects.get(
  id=stream_topic_access_group_id, realm=realm, is_system_group=True
)
````